docs/HowTo/Use/node_network_migration.md

```diff
@@ -0,0 +1,107 @@
+---
+description: Migrate node to a new IP network
+---
+
+# Migrate node to a new IP network
+
+When changing the network configuration of a node, it's recommended to follow specific steps so the new configuration it's applied and the consensus protocol continues to work as expected.
+
+!!!important
+
+    Before starting the migration, create a backup of your data directory of all Nodes.
+
+## Prerequisites
+
+- [GoQuorum installed](../GetStarted/Install.md)
+- A running [Raft network](../../Tutorials/Create-a-Raft-network.md) or [IBFT network](../../Tutorials/Create-IBFT-Network.md)
+
+## Raft
+
+### Peers with same networking configuration
+
+This applies when the node has a new networking configuration but the peers keep the same IP/Ports as before.
+
+For example:
+
+- A node changes public IP/Ports and peers keep the same public IP/Ports.
+
+- A node migrates to another private network but access peers using the same IP/Ports as before.
+
+- Node and peers are in the same private network and routing is still possible with the new networking configuration.
+
+!!!tip
```

Comment:
What is this notation?

Reply:
to create a Tip on markdown

Reply:
I mean, on the documentation. used that from other examples